proto/storage/compliance_operator_v2.proto

```diff
@@ -284,3 +284,45 @@ message ComplianceOperatorRemediationV2 {
   string enforcement_type = 7;
   string cluster_id = 8; // @gotags: search:"Cluster ID,hidden"
 }
+
+// Next Tag: 8
+message ComplianceOperatorReportSnapshotV2 {
+  string report_id = 1; // @gotags: sql:"pk,id,type(uuid)"
+  string scan_configuration_id = 2; // @gotags: search:"Compliance Scan Config ID" sql:"fk(ComplianceOperatorScanConfigurationV2:id)"
```

Comment:
What do we want this to do when the scan configuration is removed?  Do we want to keep the report snapshot or remove it?  If we there is a reason to keep it, we should add `,no-fk-constraint`.  If we want it to be deleted when the scan config is deleted then as you have it is fine.

Reply:
This is intentional, the idea is to delete the reports if the scan configuration is deleted.